pages/summary.js

```diff
@@ -2,28 +2,78 @@ import PropTypes from "prop-types";
 import { API_URL } from "../lib/constants";
 import { getQueryStringFromServerQuery } from "../lib/query-utils";
 import Breadcrumbs from "../components/breadcrumbs";
-import JsonDisplay from "../components/json-display";
+import { DataPanel, DataAreaTitle } from "../components/data-area";
+import { ButtonLink } from "../components/form-elements";
 import PagePreamble from "../components/page-preamble";
+import RegulomeVersionTag from "../components/regulome-version-tag";
+import SummaryTable from "../components/summary-table";
+import Notifications from "../components/notifications";
 
-export default function Summary({ data }) {
+export default function Summary({ data, queryString }) {
+  const total = data.total ? data.total : 0;
```

Comment:
You can say `data.total || 0` to save typing (you won’t see any measurable performance difference though; just to save typing).